apps/asap-cli/src/import/protocols/insert.ts

```diff
@@ -85,21 +85,17 @@ export default async (data: Protocol): Promise<void> => {
     title: {
       iv: data.name,
     },
-    shortText: {
-      iv:
-        data.abstract ||
-        `From Team ${data.team} and authors: ${data.authors.join(', ')}`,
-    },
     link: {
       iv: data.link,
     },
     publishDate: {
       iv: new Date(data.created).toISOString(),
     },
     text: {
-      iv: `From Team ${data.team} and authors: ${data.authors.join(
-        ', ',
-      )}. Keywords: ${data.keywords.join(', ')}.`,
+      // eslint-disable-next-line prefer-template
+      iv: `${data.abstract ? data.abstract + '\n || ' : ''}From Team ${
```

Comment:
why bother and disable the rule when you can just go with it?

Reply:
Yeah, I disabled it to prevent from having nested template strings

Reply:
yeah i think thats more effort than simply using templates but sure your call